README.rst

```diff
@@ -20,4 +20,98 @@ Python implementation of PVL (Parameter Value Language)
 Features
 --------
 
-* TODO
\ No newline at end of file
+* TODO
+
+How to use Module
+--------------------
+
+.. contents:: `Table of Contents`
+	:local:
+
+pvl.load
++++++++++
+
+General:: 
+
+This module parses an isis label from a stream and returns a dictionary 
```

Comment:
Change "This module parses an isis label from" to "This module parses an PVL compliant label from"